ops/solana/solana.go

```diff
@@ -330,16 +356,41 @@ func (d Deployer) InitOCR(keys []opsChainlink.NodeKeys) error {
 		oracles = append(oracles, map[string]string{
 			"signer":      k.OCR2OnchainPublicKey,
 			"transmitter": k.OCR2Transmitter,
-		})
-
-		operators = append(operators, map[string]string{
 			"payee":       k.OCR2Transmitter, // payee is the same as transmitter
-			"transmitter": k.OCR2Transmitter,
 		})
+
+		// operators = append(operators, map[string]string{
+		// 	"payee":       k.OCR2Transmitter, // payee is the same as transmitter
+		// 	"transmitter": k.OCR2Transmitter,
+		// })
 	}
 
-	// TODO: Should this inputs have their own struct?
+	fmt.Println("Proposing config...")
 	input := map[string]interface{}{
+		"oracles":    oracles,
+		"f":          threshold,
+		"proposalId": d.Account[Proposal],
+	}
+
+	jsonInput, err := json.Marshal(input)
+	if err != nil {
+		return err
+	}
+
+	err = d.gauntlet.ExecCommand(
+		"ocr2:propose_config",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("proposalId", d.Account[Proposal]),
+		d.gauntlet.Flag("input", string(jsonInput)),
+		d.Account[OCRFeed],
+	)
+
+	if err != nil {
+		return errors.Wrap(err, "setting OCR 2 config failed")
```

Comment:
```suggestion
		return errors.Wrap(err, "'ocr2:propose_config' call failed")
```

If this is surfaced through an original error, maybe even is unnecessary to wrap.